functions.php

```diff
@@ -0,0 +1,161 @@
+<?php
+require_once 'dbConn.php';
+
+/*
+ * Is used to select data from the database and return it as a multidimensional array, designed specifically for the about me content
+ *
+ * @param - PDO - $db - this is where the $db var should be used otherwise the function can't access the database(scope)
+ * @param - string - $table - used to choose which table you want from the db, should be used in double quotes with backticks.
+ * @param - string - $field - which field you want from the table, in double quotes with backticks
+ *
+ * @return - an array - should be an array of the field that's picked.
+ */
+function retrieveData (PDO $db, string $table, string $field): array
+{
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $query = $db->prepare("SELECT $field FROM $table;");
+    $query->execute();
+    $result = $query->fetchAll();
+    if (is_array($result) == true) {
```

Comment:
`$result` will always be an array, it just might be empty. you should use `empty()` instead